app/components/view-section-table-row/template.njk

```diff
@@ -15,6 +15,29 @@
     {% endverbatim %}
   </code></pre>
 
+  {% set commonAttributes %} data-test-id="view-section-table-row-{{ params.questionId }}" class="nhsuk-width-container bc-c-section-table-row"{%- endset -%}
+
+  <h3>Generated HTML</h3>
+  <pre><code>
+    {{- "
+      <div class='bc-c-title-block'>
+        <h3 class='nhsuk-u-margin-bottom-2'>Example section heading</h3>
+      </div>
+        <div data-test-id='view-section-table-row-some-question-id' class='nhsuk-width-container bc-c-section-table-row'>
+        <div data-test-id='view-section-table-row-horizontal' class='nhsuk-grid-row'>
+          <div data-test-id='view-section-table-row-title' class='nhsuk-u-font-weight-bold nhsuk-grid-column-one-half'>
+            Some question title
+          </div>
+          <div data-test-id='view-section-table-row-component' class='nhsuk-grid-column-one-half'>
+            <div>
+              Some inner component
+            </div>
+          </div>
+        </div>
+      </div>
+    " }}
+  </code></pre>
```

Comment:
delete

Reply:
we consistently do this elsewhere so why not here?

Reply:
we decided to take it out. We've removed it elsewhere as we have gone along